docs/dashboards/custom-chart.md

```diff
@@ -0,0 +1,164 @@
+# Custom Chart
+
+The Custom Chartm Block is an advanced Block that allows you to create custom data visualization by utilizing [Vega-Lite](https://vega.github.io/vega-lite/). If you would like a specific visualization that's not already a Losant Dashboard Block, the Custom Chart Block gives you the ultimate support for a wide array of visualization types.
+
+![Data Table Scatter Plot](../images/workflows/custom-chart-data-table-scatter-plot.png "Data Table Scatter Plot")
+
+## Configuration
+
+Configuring the Custom Chart Block is broken up into three sections ...
+
+### Choosing an Application
```

Comment:
(optional) I don't see "Choosing an Application" in any other dashboard block doc